README.md

```diff
@@ -2,66 +2,106 @@
 
 A bunch of English words in JSON format.
 
-Words obtained from the [SCOWL][] project. See the `Copyright` file.
+Words obtained from the [SCOWL][] project. See the `Copyright (SCOWL)` file.
 
 [SCOWL]: http://wordlist.aspell.net/
 
-You can import the module as an ES module or using CommonJS:
+## Usage
+
+The simplest way to use this package is to install it with NPM and to import the default export from “wordlist-english”:
+
+```
+npm install wordlist-english
+```
+
+```js
+import englishWords from 'wordlist-english';
+
+englishWords[2**12] // => 'auditor'
+englishWords[2**13] // => 'bridle'
+englishWords[2**14] // => 'crude'
+englishWords[2**15] // => 'highways'
+englishWords[2**16] // => 'soar'
+```
+
+However, if you are using the words in a web application, it may be wiser to use a [dynamic import][] so you can more quickly serve your web page—i.e., without waiting for a large list of words to download as part of your main JavaScript bundle.
+
+[dynamic import]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import
+
+```js
+const englishWords = (await import('wordlist-english')).default;
+// The words won’t be available synchronously, but depending on how your
+// application works it might be okay or preferable to load the words in the
+// background or on-demand.
+```
+
+## Available Lists
+
+To see examples of the types of words in each list, see [SAMPLES.md](SAMPLES.md).
+
+### Default List (270kb gzipped)
+
+If you simply want a big list of English words that most English speakers would possibly say or write (all dialects included: American, Australian, British, Canadian), then import the package’s index file:
 
 ```js
-import wordlist from 'wordlist-english'; // ES Modules
-var wordlist = require('wordlist-english'); // CommonJS
+import englishWords from 'wordlist-english';
+const englishWords = (await import('wordlist-english')).default;
```

Comment:
@jacksonrayhamilton Just a thought, but I think it might be visually clearer to have the dynamic import example in a separate (still adjacent) code block from the static import example. Otherwise, it makes it seem like both are necessary to write. when really it's an either-or thing. For example, rewrite as:

```js
import englishWords from 'wordlist-english';
```

```js
const englishWords = (await import('wordlist-english')).default;
```

Reply:
I deleted all the extra dynamic import examples.  One was enough.